src/lib/mappers/user.js

```diff
@@ -15,7 +16,7 @@ const dbToModel = data => {
 
 const pojoToModel = dbToModel;
 
-const modelToDb = model => model.toJSON();
+const modelToDb = model => isObject(model) ? model.toJSON() : null;
```

Comment:
I'm wondering if the `:null` on the end should be `:{}` (So that the returned type is always an object)

If the null is intentional, ignore this comment.

Reply:
This one is intentional - it gets mapped to DB column which is either a jsonb containing the user or null